### PR TITLE
Add Information Security Corp. as member

### PR DESCRIPTION
**Pull request by markowitz-isc**

### CODE REVIEW

**Review by vanbroup — CHANGES_REQUESTED** [review]

**Comment by vanbroup** [comment]
Thanks for the pull request, I have split the PR into one to fix the spelling error you found and one to add Information Security Corp. as a member.

Please update the logo as this is currently containing some additional whitespace and actual text instead of rasterized text that shows the same irrespective of the device that is loading the logo.

**Comment by vanbroup** [comment]
Looking forward to the update

File: assets/images/members/infoseccorp/infoseccorp.svg
[the remark this answers is not in the document]
Comment (vanbroup) [reply assets/images/members/infoseccorp/infoseccorp.svg:1]: Any updates on this?

**Comment by vanbroup** [comment]
@markowitz-isc looking forward to your updates so that we can close this PR

**Comment by markowitz-isc** [comment]
@vanbroup Sorry, Paul. Working on this now...

**Comment by vanbroup** [comment]
> @vanbroup Hopefully one of these will work for your needs. Thanks!
> 
> ISClogo-Corp_white ISClogo-Corp_Black

Unfortunately this is not a vector image like an SVG but a PNG (pixel) image.

**Comment by blackbird1999** [comment]
[logos.zip](https://github.com/pkic/pkic.org/files/12549436/logos.zip)
@vanbroup The original Adobe Illustrator (.ai) and its exported SVG are in the attached zip. Thank you.

**Comment by blackbird1999** [comment]
ISClogo-Corp_Spread_Out_Black - Raster
And a slightly different SVG where I've, again, told Illustrator to rasterize the text...

**Comment by blackbird1999** [comment]
ISClogo-Corp_Spread_Out_white - Raster
And the white one...

**Comment by blackbird1999** [comment]
ISClogo-Corp_Spread_Out_white - Raster
And the continuing fight with Illustrator.

**Comment by vanbroup** [comment]
I have updated the logo based on the AI file, hope that this one looks like the original:

https://github.com/pkic/pkic.org/blob/1ae9cff9b8b69a894dfb97cd66038108f6b9150c/assets/images/members/infoseccorp/infoseccorp.svg

**Review by vanbroup — APPROVED** [review]

**Comment by vanbroup** [comment]
Merging for preview in isc branch

**Pull request merged**